system/prometheus-vmware/requirements.yaml

```diff
@@ -0,0 +1,12 @@
+dependencies:
```

Comment:
The dependencies block can go into the Chart.yaml as Helm does not use `requirements.yaml` and `requirements.lock` anymore. ([Example](https://github.com/sapcc/helm-charts/blob/master/system/kube-monitoring-metal/Chart.yaml#L7))